sklearn/ensemble/gradient_boosting.py

```diff
@@ -1296,6 +1297,14 @@ class GradientBoostingClassifier(BaseGradientBoosting, ClassifierMixin):
         of the input variables.
         Ignored if ``max_leaf_nodes`` is not None.
 
+    criterion : string, optional (default="friedman_mse")
```

Comment:
Sorry for being late to the party, but this should have a ``versionadded``, right?